spec/manual/find_executable_spec.cr

```diff
@@ -0,0 +1,58 @@
+# Verifies that find_executable's specs match the behavior of Process.run.
+# This doesn't actually test find_executable, only takes all the test cases
+# directly from spec/std/process/find_executable_spec.cr and checks that
+# *they* match what the OS actually does when finding an executable for the
+# purpose of running it.
+
+require "spec"
+require "digest/sha1"
+require "../support/env"
+require "../support/tempfile"
+require "../std/process/find_executable_spec"
+
+describe "Process.run" do
+  test_dir = Path[SPEC_TEMPFILE_PATH] / "manual_find_executable"
+  base_dir = Path[test_dir] / "base"
+  path_dir = Path[test_dir] / "path"
+
+  around_all do |all|
+    Dir.mkdir_p(test_dir)
+
+    exe_names, non_exe_names = FIND_EXECUTABLE_TEST_FILES
+    exe_names.each do |name|
+      src_fn = test_dir / "self_printer.cr"
+      exe_fn = test_dir / "self_printer.exe"
+      File.write(src_fn, "print #{name.inspect}")
+      Process.run("crystal", ["build", "-o", exe_fn.to_s, src_fn.to_s])
```

Comment:
I think the wrapper should be used here. crystal might not be on the path.

```suggestion
      Process.run("bin/crystal", ["build", "-o", exe_fn.to_s, src_fn.to_s])
```

Someday I will automate the manual specs in test-ecosystem probably...